Mathlib/Combinatorics/SimpleGraph/ConcreteColorings.lean

```diff
@@ -7,6 +7,7 @@ import Mathlib.Combinatorics.SimpleGraph.Coloring
 import Mathlib.Combinatorics.SimpleGraph.Hasse
 import Mathlib.Data.Nat.Parity
 import Mathlib.Data.ZMod.Basic
+import Mathlib.Logic.Lemmas
```

Comment:
Are you sure you need those new imports?

Reply:
It's importing `injective_not`, if I delete it I get the error: `unknown identifier 'injective_not'`